src/test/java/uk/ac/cam/cl/dtg/isaac/api/IsaacIntegrationTest.java

```diff
@@ -394,4 +397,23 @@ protected HttpServletRequest createRequestWithCookies(final Cookie[] cookies) {
     expect(request.getCookies()).andReturn(cookies).anyTimes();
     return request;
   }
+
+  protected HttpServletRequest prepareUserRequest(String userEmail, String userPassword) {
```

Comment:
This (and the other new method) is only used in one test file, so it should be moved there.